homeassistant/components/dyson/air_quality.py

```diff
@@ -0,0 +1,146 @@
+"""
+Support for Dyson Pure Cool Air Quality Sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/dyson/
```

Comment:
We don't add the url in the docstring anymore. Just keep the very first line.

Reply:
Removed.